buglist-link.js

```diff
@@ -0,0 +1,37 @@
+var bugs = [];
+
+function getCommonLandings() {
+	//look for Bug # on page
+	let result;
+	let re = /Bug ([0-9]+)/ig;
+	while ((result = re.exec(document.body.innerText)) !== null) {
+		bugs.push(result[1]);
+	}
+	//consolidate Bug #
+	bugs = new Set(bugs);
+	
+	if (bugs.size === 0) {
+		return;
+	}
+
+	var title = document.getElementsByClassName('title');
```

Comment:
``const titlebar = document.querySelector("div.title");`` gets only the first one and if it's null then return